### PR TITLE
Add option to server keys command that creates formatted key files without a device being connected

### PR DESCRIPTION
**Pull request by JamesHagerman**
Story details: https://app.clubhouse.io/particle/story/48657

### CODE REVIEW

**Comment by busticated** [comment]
@JamesHagerman let's pair on adding a test - ping me when you are ready to dive in :+1:

**Comment by monkbroc** [comment]
I'll let you James and Mirande pair on this

**Comment by JamesHagerman** [comment]
We paired on this and another cli pr! I'll add tests to this now.

File: test/e2e/keys.e2e.js [hunk 1]
```diff
@@ -47,6 +69,148 @@ describe('Keys Commands [@device]', () => {
...
+		}).timeout(5 * 60 * 1000);
+
+		it('Fails to switch server public keys when device is not in DFU mode', async () => {
+			const { stdout, stderr, exitCode } = await cli.run(['keys', 'server']);
+
+			expect(stdout.split('\n')).to.include.members(dfuInstructions);
+			expect(stderr).to.equal('');
+			expect(exitCode).to.equal(1);
+		});
+	});
```
Comment (busticated) [inline test/e2e/keys.e2e.js:214]: @JamesHagerman added these so at least we have a examples that you can build on and some supporting tooling but they're slower and flakier than i'd like - mostly due to the time it takes for a device to connect :(

one thing it did highlight is that the `particle keys server --deviceType photon` command form doesn't appear to do anything. it should probably be a usage error ([example](https://github.com/particle-iot/particle-cli/commit/6cbb33eab9c308cbc5a4135834eb2a3a808cb443#diff-1c5e41ba62634f0609f01bdf97c1a93fR155))..?

**Comment by busticated** [comment]
CI is failing due to coveralls 🤦 

```
/Users/travis/build/particle-iot/particle-cli/node_modules/coveralls/bin/coveralls.js:19
      throw err;
      ^
Bad response: 422 {"message":"service_job_id (678673063) must be unique for Travis Jobs not supplying a Coveralls Repo Token","error":true}
```

...which is why i did this https://github.com/particle-iot/particle-cli/pull/571/commits/19c43d35040277b5cc0cc4c70f8191ee15c84f5f

i'll get it all cleaned up post-merge 👍

**Pull request merged**